sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroSchemaIOProvider.java

```diff
@@ -109,12 +107,12 @@ public Schema schema() {
     }
 
     @Override
-    public PTransform<PCollection<Row>, POutput> buildWriter() {
+    public PTransform<PCollection<Row>, WriteFilesResult<?>> buildWriter() {
```

Comment:
This looks good, here the change has less issues since this class is `@Internal`
CC @TheNeuralBit for awarenes.

Reply:
Ack, thank you